plugins/PostProcessingPlugin/scripts/ChangeAtZ.py

```diff
@@ -279,6 +281,36 @@ def getSettingDataString(self):
                     "minimum_value_warning": "15",
                     "maximum_value_warning": "255",
                     "enabled": "j1_Change_fanSpeed"
+                },
+                "k1_Change_acceleration":
+                {
+                    "label": "Change acceleration",
+                    "description": "Select if you want to change max. acceleration. The setting will be overriden when you also use Cura profile acc. settings. Works only with layer number, not height.",
+                    "type": "bool",
+                    "default_value": false
+                },
+                "k2_acceleration":
```

Comment:
Looking at other parameters, I expected this to be named k3_ and the next one k4_